libs/utils/src/utils/ethersFallback.ts

```diff
@@ -2,19 +2,22 @@
 // ethers.js getFeeData returns maxFeePerGas as undefined,  might need to use gasPrice some how, then fetch the L1 amount
 // https://community.optimism.io/docs/developers/build/transaction-fees/#displaying-fees-to-users
 
-import { ethers } from 'ethers';
-import { getProcessingGasLimit } from './gas';
+// import { ethers } from 'ethers';
+import { fetchFeeData, getProcessingGasLimit } from './gas';
+import { ValidNetwork } from '@daohaus/keychain-utils';
 
 export const getGasCostEstimate = async (
   gasLimit: number | string,
   rpcUrl: string,
   chainId: string
 ): Promise<number | undefined> => {
-  const ethersProvider = new ethers.providers.JsonRpcProvider(rpcUrl);
-  const feeData = await ethersProvider.getFeeData();
+  // const ethersProvider = new ethers.providers.JsonRpcProvider(rpcUrl);
```

Comment:
We can log out both results on a few proposals to test and see if we're getting the same number.